src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -4698,7 +4698,7 @@ private static void TraceFilters(CmdletProviderContext context)
                 // Trace the filter
                 s_pathResolutionTracer.WriteLine("Filter: {0}", context.Filter ?? string.Empty);
 
-                if (context.Include != null)
+                if (context.Include != null && context.Include.Count > 0)
```

Comment:
Maybe `if (context?.Include.Count > 0)`

Reply:
It looks like `if (context.Include?.Count > 0)` would be closer to the original framing here.